vizro-core/docs/pages/user-guides/custom-figures.md

```diff
@@ -0,0 +1,105 @@
+# How to create custom Figures
+
+This guide explains how to create custom figures, which is useful when you need a figure that reacts to controls and actions.
```

Comment:
If you've used "custom figures" here, why capitalise it above in the title?

I think it needs to be consistent, but am not sure which you'd want to do -- probably lower case throughout? See above suggestion if so, and accept.